src/main/java/io/strimzi/kafka/bridge/quarkus/CustomMeterRegistryProducer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.quarkus;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.prometheus.PrometheusConfig;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import io.micrometer.prometheus.PrometheusNamingConvention;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * Used for scraping and reporting metrics in Prometheus format
```

Comment:
I will add more details why we need a custom registry. So explaining we are gathering both Quarkus HTTP server provided metrics + our own Kafka metrics through the JMX collector.

Reply:
Okay